keras_cv/models/segmentation/deeplab.py

```diff
@@ -89,22 +78,22 @@ def __init__(
 
         if input_shape[0] is None and input_shape[1] is None:
             raise ValueError(
-                "Input shapes for both the backbone and DeepLabV3 are `None`."
+                "Input shapes for both the backbone and DeepLabV3 cannot be `None`. "
+                f"Received: input_shape={input_shape} and "
+                f"backbone.input_shape={backbone.input_shape[1:]}"
             )
 
         height = input_shape[0]
         width = input_shape[1]
 
-        x = inputs
-
-        feature_map = backbone(x)
+        feature_map = backbone(inputs)
         if spatial_pyramid_pooling is None:
             spatial_pyramid_pooling = SpatialPyramidPooling(
                 dilation_rates=[6, 12, 18]
             )
 
         output = spatial_pyramid_pooling(feature_map)
-        output = tf.keras.layers.UpSampling2D(
```

Comment:
We've made this mistake in a lot of places in KerasCV, so I opened #1527 to clean this up